### PR TITLE
upload baize auth table

### PR DESCRIPTION
**Pull request by justedennnnn**
See preview: https://deploy-preview-4985--daocloud-docs.netlify.app/ghippo/permissions/baize

### CODE REVIEW

**Review by windsonsea — COMMENTED** [review]
See preview: 
https://deploy-preview-4985--daocloud-docs.netlify.app/ghippo/permissions/baize

**Review by windsonsea — APPROVED** [review]

File: docs/zh/docs/ghippo/permissions/baize.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+hide:
+  - toc
+---
+
+# 智能算力权限说明
+
+[智能算力](../../amamba/intro/index.md)支持四种用户角色：
```
Comment (windsonsea) [inline docs/zh/docs/ghippo/permissions/baize.md:8]:
```suggestion
[智能算力](../../baize/intro/index.md)支持四种用户角色：
```

File: docs/zh/docs/ghippo/permissions/baize.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+- Baize Owner
+- Workspace Admin
+- Workspace Editor
+- Workspace Viewer
+
+每种角色具有不同的权限，具体说明如下。
+
+| **菜单对象**  | **操作**         | **Admin / Baize Owner** | **Workspace Admin** | **Workspace Editor** | **Workspace Viewer** |
```
Comment (samzong) [inline docs/zh/docs/ghippo/permissions/baize.md:17]: Admin / Baize Owner， 这里的 Admin 要补充下介绍.

**Comment by samzong** [comment]
漏一个 任务分析的删除 权限点.
image

File: docs/zh/docs/ghippo/permissions/baize.md (hunk 2 above)
Comment (justedennnnn) [reply docs/zh/docs/ghippo/permissions/baize.md:17]: done

**Comment by justedennnnn** [comment]
> 漏一个 任务分析的删除 权限点. image

done

File: docs/zh/docs/ghippo/permissions/baize.md (hunk 1 above)
Comment (windsonsea) [reply docs/zh/docs/ghippo/permissions/baize.md:8]: 啊额，怎么回滚了

**Pull request merged**